src/sass/_hero.scss

```diff
@@ -0,0 +1,194 @@
+.section--header {
+  background-color: getColor('primary-pink');
+}
+
+.hero-header {
+  padding-top: 6px;
+  height: 100vh;
+  position: relative;
+}
+
+.hero-header__link {
+  color: getColor('accent');
+}
+
+.hero-header__svg {
+  fill: currentColor;
+}
+
+.hero-header__img {
+  position: absolute;
+  bottom: 0;
+  right: 12px;
+  z-index: 1;
+}
+
+@media screen and (min-width: 768px) {
+  .hero-header {
+    padding-top: 37px;
+  }
+}
+
+.hero-header__top {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 137px;
+}
+
+.hero-header__bottom {
+  position: absolute;
+  z-index: 2;
+  bottom: 160px;
+}
+
+.hero-header__title-div {
+  width: 218px;
+  margin-bottom: 19px;
+}
+
+.hero-header__h1 {
+  font-family: getFont('font-header');
+  font-weight: getFontWeigh('font-regular');
+  font-size: 48px;
+  line-height: 0.97;
+  text-transform: uppercase;
+  color: getColor('accent');
+}
+
+.hero-header__h2 {
+  font-family: getFont('font-header');
+  font-weight: getFontWeight('font-regular');
+  font-size: 26px;
+  line-height: 1.15;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  color: getColor('background-white');
+}
+
+.hero-header__buttons {
+  display: flex;
+}
+
+.button__header {
```

Comment:
nie wiem, czy nie lepiej ustawić szerokość i wysokość buttona za pomocą paddingów, ale to sugestia tylko